MovieQuiz/Presentation/ResultAlertPresenter.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+
+class ResultAlertPresenter {
+    private let title: String
+    private let message: String
+    private let controller: UIViewController
+    private let actionHandlder: ((UIAlertAction) -> Void)?
+
+    var currentQuestionIndex: Int = 0
+    var counterCorrectAnswers: Int = 0
+     var numberOfQuizGames: Int = 0
+     var recordCorrectAnswers: Int = 0
+     var recordDate = Date()
+     var averageAccuracy: Double = 0.0
+     var allCorrectAnswers: Int = 0
```

Comment:
```suggestion
    var numberOfQuizGames: Int = 0
    var recordCorrectAnswers: Int = 0
    var recordDate = Date()
    var averageAccuracy: Double = 0.0
    var allCorrectAnswers: Int = 0
```

Советую пользоваться форматтером: выделите весь код и зажмите control + I